common/src/View/MapViewBase.cpp

```diff
@@ -947,11 +948,15 @@ namespace TrenchBroom {
             Model::Node* newBrushParent = findNewParentEntityForBrushes(nodes);
             Model::Node* currentGroup = document->editorContext().currentGroup();
             Model::Node* newGroup = findNewGroupForObjects(nodes);
+            Model::Node* mergeGroup = findGroupToMergeGroupsInto(document->selectedNodes());
             
             wxMenu menu;
             menu.SetEventHandler(this);
             menu.Append(CommandIds::MapViewPopupMenu::GroupObjects, "Group");
             menu.Append(CommandIds::MapViewPopupMenu::UngroupObjects, "Ungroup");
+            if (mergeGroup != nullptr) {
```

Comment:
I wonder if it's better for the UI to show the menu item, but disable it if merging is not possible. Might make it easier for the users muscle memory to have the same menu structure.

Reply:
Agreed, just made that change.